src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1418,6 +1418,102 @@ def aks_upgrade(cmd, client, resource_group_name, name, kubernetes_version, no_w
     return sdk_no_wait(no_wait, client.create_or_update, resource_group_name, name, instance)
 
 
+def aks_use_dev_spaces(cmd, client, cluster_name, resource_group_name, space_name='default', parent_space_name=None):  # pylint: disable=line-too-long
+    """
+    Use Azure Dev Spaces with a managed Kubernetes cluster.
```

Comment:
Should this be prefixed with (PREVIEW)?

Reply:
This is just a summary of the method. It is not accessible to the user.